pymc3/distributions/continuous.py

```diff
@@ -722,6 +722,24 @@ def _distr_parameters_for_repr(self):
         return ["mu", "sigma", "lower", "upper"]
 
 
+class PyMC3HalfNormalRV(RandomVariable):
```

Comment:
This `PyMC3*` naming scheme isn't good.  The `RandomVariable`s defined in PyMC3 should be generally applicable.  Furthermore, we should expect that other libraries will use these as the canonical Aesara `RandomVariable`s&mdash;when these variables aren't already in `Aesara`.

Reply:
Agree. This was just to distinguish the Aesara / PyMC variations when they were incompatible.